src/pages/journal-vouchers/index.js

```diff
@@ -122,17 +124,36 @@ const JournalVoucher = () => {
     toast.success('Record Deleted Successfully')
   }
 
+  const onApply = ({ search, rpbParams }) => {
+    if (!search && rpbParams.length === 0) {
+      clearFilter('params')
+    } else if (!search) {
+      filterBy('params', rpbParams)
+    } else {
+      filterBy('qry', search)
+    }
+    refetch()
+  }
+
+  const onSearch = value => {
+    filterBy('qry', value)
+  }
+
+  const onClear = () => {
+    clearFilter('qry')
+  }
+
   return (
     <VertLayout>
       <Fixed>
-        <GridToolbar
+        <RPBGridToolbar
           onAdd={add}
           maxAccess={access}
-          onSearch={search}
-          onSearchClear={clear}
-          labels={_labels}
-          inputSearch={true}
-        />{' '}
+          onApply={onApply}
+          onSearch={onSearch}
+          onClear={onClear}
+          reportName={'GLTR'}
```

Comment:
check if the reportName in all reports should be written manually or as label

Reply:
will check with nora
these are not used many times